qiskit_experiments/library/stark/ramsey_xy.py

```diff
@@ -0,0 +1,319 @@
+        self,
+        qubit: int,
+        stark_amp: float,
+        backend: Optional[Backend] = None,
+        **experiment_options,
+    ):
+        """Create new experiment.
+
+        .. note::
+
+            .. _stark_amplitude:
+
+            The `stark_channel` in the experiment options may be necessary supplied.
+            By default, the Stark tone is applied to the same channel with the qubit drive
+            with frequency modulation. This frequency modulation might update the
+            channel frame, which accumulates unwanted phase against the frequency difference
+            between the qubit drive and Stark tone frequency in addition to
+            the qubit Stark shift. You can use a dedicated Stark drive channel if available,
+            otherwise you may want to reuse a control channel associated with the qubit.
+            This depends on the architecture of your quantum device.
+
+        Args:
+            qubit: Index of qubit.
+            stark_amp: A single float parameter to represent the magnitude of Stark tone
+                and the sign of expected Stark shift. See :ref:`stark_amplitude` for details.
+            backend: Optional, the backend to run the experiment on.
+            experiment_options: Extra experiment options. See self.experiment_options.
+        """
+        self._timing = None
+
+        super().__init__(qubits=[qubit], analysis=RamseyXYAnalysis(), backend=backend)
+        self.set_experiment_options(stark_amp=stark_amp, **experiment_options)
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        """Default experiment options.
+
+        Experiment Options:
+            stark_amp (float): A single float parameter to represent the magnitude of Stark tone
+                and the sign of expected Stark shift. See :ref:`stark_amplitude` for details.
+            stark_channel (PulseChannel): Pulse channel to apply Stark tones.
+                If not provided, the same channel with the qubit drive is used.
+            stark_freq_offset (float): Offset of Stark tone frequency from the qubit frequency.
+                This must be greater than zero not to apply Rabi drive.
+            stark_sigma (float): Gaussian sigma of the rising and falling edges
+                of the Stark tone, in seconds.
+            stark_risefall (float): Ratio of sigma to the duration of
+                the rising and falling edges of the Stark tone.
+            min_freq (float): Minimum frequency that this experiment is guaranteed to resolve.
+                Note that fitter algorithm :class:`.RamseyXYAnalysis` of this experiment
+                is still capable of fit experiment data with further lower frequency.
+            max_freq (float): Maximum frequency that this experiment can resolve.
+            delays (list[float]): The list of delays in set that will be scanned in the
+                experiment. If not set, then evenly spaced delays with interval
```

Comment:
Should there be a reference to the `delays` method which gives more description?